code/modules/virus2/effect/effect.dm

```diff
@@ -212,9 +211,79 @@
 			for(var/i = 0 to multiplier)
 				new/mob/living/simple_animal/bee(get_turf(mob))
 
+/datum/disease2/effect/hyronalinis
+	name = "Hyronalinis"
+	stage = 1
+
+/datum/disease2/effect/hyronalinis/activate(var/mob/living/carbon/mob)
+	to_chat(mob, "<span class = 'notice'>You gained radiation resistance.</span>")
+	if (prob(85))
+		mob.reagents.add_reagent(HYRONALIN, 1)
+
 
-////////////////////////STAGE 2/////////////////////////////////
+/datum/disease2/effect/soreness
+	name = "Soreness"
+	stage = 1
+
+/datum/disease2/effect/soreness/activate(var/mob/living/carbon/mob)
+	to_chat(mob, "<span class = 'notice'>You feel a little sore.</span>")
 
+/datum/disease2/effect/salutaretin
+	name = "Salutaretin Syndrome"
+	stage = 1
+
+/datum/disease2/effect/salutaretin/activate(var/mob/living/carbon/mob)
+	if(..())
+		return 1
+
+	if(mob.stat || mob.health < 90 || mob.getBrainLoss() >= 10)
+		return 1
+
+	var/list/nearest_player = null
+	for(var/mob/living/L in view(mob))
+		if(L == mob)
+			continue
+		if(L.stat)
+			continue
+		if(nearest_player && get_dist(L,mob)>=get_dist(nearest_player,mob))
+			continue //We already have a closer living target
+		if(ishuman(L))
+			var/mob/living/carbon/human/H = L
```

Comment:
casting to human is not necessary, all you're doing is assigning `L` to `nearest_player`, so just do
```suggestion
			nearest_player = L
```
or combine the two and
```
if(ishuman(L) || isrobot(L))
	nearest_player = L
```